CHANGELOG.md

```diff
@@ -2,6 +2,17 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+<a name="0.9.1"></a>
```

Comment:
There is no need to manually update this file. I use an automatic process for that. (However I have to research again how that works...)

Reply:
I had followed the steps in [this file](https://github.com/StefH/angular-odata-es5/blob/master/way-of-working.txt), which automatically added these changes. I've reverted them for now so that you can take care of it once the changes are merged and ready to release.